beacon-chain/db/kv/lightclient_test.go

```diff
@@ -213,19 +228,20 @@ func TestStore_LightClientUpdate_CanSaveRetrieve(t *testing.T) {
 		require.NoError(t, err)
 		require.DeepEqual(t, update, retrievedUpdate, "retrieved update does not match saved update")
 	})
-	t.Run("Capella", func(t *testing.T) {
-		update, err := createUpdate(t, version.Capella)
+	t.Run("Bellatrix", func(t *testing.T) {
```

Comment:
Looks like a great opportunity to make this iterative...

```
for _, v := range version.All() {
  t.Run(version.String(v), func(t *testing.T) {
    update, err := createUpdate(t, v)
   // the rest of the code...
  }
}
```

Then you can delete all of these individual code blocks since it's all identical besides the version argument in createUpdate